<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBoxInputParam">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>621</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Model Input Parameters (will not be fitted)</string>
   </property>
   <widget class="QLabel" name="labelInputParam1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>R (Motion blur correction)</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBoxInputParam1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="maximum">
     <double>0.250000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>0.166700000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxModel1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Brownian Diffusion</string>
   </property>
   <widget class="QLineEdit" name="lineEditParam1_1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditParam2_1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelParam1_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>D (m^2/s)</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelParam2_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>delta (m)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRelLikelihood_1">
    <property name="geometry">
     <rect>
      <x>421</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBIC_1">
    <property name="geometry">
     <rect>
      <x>421</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelRelLikelihood_1">
    <property name="geometry">
     <rect>
      <x>341</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rel. Likelihood</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBIC_1">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>BIC</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditParam1Error_1">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditParam2Error_1">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxModel1_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>621</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Confined Diffusion</string>
   </property>
   <widget class="QLineEdit" name="lineEditParam1_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditParam2_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelParam1_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>D_micro (m^2/s)</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelParam2_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>delta (m)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRelLikelihood_2">
    <property name="geometry">
     <rect>
      <x>421</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBIC_2">
    <property name="geometry">
     <rect>
      <x>421</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelRelLikelihood_2">
    <property name="geometry">
     <rect>
      <x>341</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rel. Likelihood</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBIC_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>BIC</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditParam1Error_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditParam2Error_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditParam3Error_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditParam3_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelParam3_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>tau (s)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxModel1_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>621</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Hopping Diffusion</string>
   </property>
   <widget class="QLineEdit" name="lineEditParam1_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditParam2_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelParam1_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>D_macro (m^2/s)</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelParam2_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>D_micro (m^2/s)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRelLikelihood_3">
    <property name="geometry">
     <rect>
      <x>421</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBIC_3">
    <property name="geometry">
     <rect>
      <x>421</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelRelLikelihood_3">
    <property name="geometry">
     <rect>
      <x>341</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rel. Likelihood</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBIC_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>BIC</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditParam1Error_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditParam2Error_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditParam3Error_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditParam3_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelParam3_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>delta (m)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditParam4Error_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelParam4_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>tau (s)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditParam4_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonSetRange">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>858</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Range</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxRange">
   <property name="geometry">
    <rect>
     <x>381</x>
     <y>858</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAnalyze">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>460</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Analyze</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxLogPlot">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>860</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Log Scale</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClipboard">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Results to Clipboard</string>
   </property>
  </widget>
  <widget class="ModelFitWidget" name="plot">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>621</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxUseInitial">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>460</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Use parameters as initial guesses</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMaxIt">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>460</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1000</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMaxIt">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>460</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Max. fit iterations</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDocumentation">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>860</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;b&gt;&lt;a href=&quot;https://eggeling-lab-microscope-software.github.io/TRAIT2D/analysis_gui.html#adc-analysis-tab&quot;&gt;Documentation&lt;/a&gt;&lt;/b&gt;</string>
   </property>
   <property name="openExternalLinks">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ModelFitWidget</class>
   <extends>QGraphicsView</extends>
   <header>trait2d_analysis_gui.tab.msd</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
